### DISCUSSION TITLE
Is it possible to output inside style tags instead of an external css file?

### DISCUSSION THREAD

**Discussion by dungle-scrubs** [tailwindlabs/tailwindcss#1553] [General]
Hi,

Is it possible to output the generated css file into a block of internal css between two style tags in an html file instead of generating an external file?

Thanks

**Comment by royvanv** [comment]
This depends on which tools you are using. You should probably just create a file and then inject that into your HTML.

**Comment by dungle-scrubs** [comment]
@royvanv Thanks, I misunderstood how Tailwind is meant to work when writing that.  But for anyone else looking for a similar answer, using webpack with postcss and style-loader, I am able to do this.

```
{
        test: /\.css$/,
        exclude: /node_modules/,
        use: [
          {loader: 'style-loader', options: { injectType: 'singletonStyleTag' }},
          {loader: "css-loader", options: {importLoaders: 1,}},
          'postcss-loader'
        ]
      }
```
